core/src/test/java/org/apache/iceberg/rest/RESTCatalogAdapter.java

```diff
@@ -568,4 +649,9 @@ private static TableIdentifier identFromPathVars(Map<String, String> pathVars) {
     return TableIdentifier.of(
         namespaceFromPathVars(pathVars), RESTUtil.decodeString(pathVars.get("table")));
   }
+
+  private static TableIdentifier viewIdentFromPathVars(Map<String, String> pathVars) {
```

Comment:
Since this is in tests, you could just combine the table and view versions by renaming the variable to `name` right?

Reply:
yep, good idea